### PR TITLE
builder/dockerfile: improve error message about build stage name

### PR DESCRIPTION
**Pull request by AkihiroSuda**
PTAL @tonistiigi 

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

**Comment by tonistiigi** [comment]
Comparing this to an invalid reference, the error message just says "invalid reference format". I would assume that if anyone sees this error they already know what symbol caused the conflict. Maybe a docs addition instead.

**Comment by AkihiroSuda** [comment]
Removed regex pattern from the PR. PTAL

File: builder/dockerfile/dispatchers.go [hunk 1]
```diff
@@ -211,7 +211,7 @@ func from(b *Builder, args []string, attributes map[string]bool, original string
 			return errors.Errorf("invalid name for build stage: %q, name can't start with a number or contain symbols", ctxName)
 		}
 	} else if len(args) != 1 {
-		return errExactlyOneArgument("FROM")
+		return errors.New("FROM requires exactly one or three arguments")
```
Comment (thaJeztah) [inline builder/dockerfile/dispatchers.go]: Perhaps _either_ one or three arguments

```
FROM requires either one or three arguments
```

Perhaps (as a follow up) we should enhance the error message in the CLI, and provide some concrete examples (`FROM foo:bar` or `FROM foo:bar AS baz`) - just thinking out loud here

**Comment by AkihiroSuda** [comment]
done @thaJeztah

**Review by vdemeester — APPROVED** [review]
LGTM 🦁 
/cc @thaJeztah @dnephin @tonistiigi

**Review by dnephin — APPROVED** [review]
LGTM

**Pull request merged**